docs/testing/platform-limits.md

```diff
@@ -44,7 +44,7 @@ If you're encountering errors when dealing with large datasets, try using multip
 
 ### Power Automate
 
-When using Office Scripts with Power Automate, each user is limited to **200 calls per day**. This limit resets at 12:00 AM UTC.
+When using Office Scripts with Power Automate, each user is limited to **400 executions of the Run Script action per day**. This limit resets at 12:00 AM UTC.
```

Comment:
Thanks for this clarification. We do need avoid using "executions", as that does not translate well. I think your additional text clarifies "calls" appropriately. What do you think about this suggestion?
```suggestion
When using Office Scripts with Power Automate, each user is limited to **400 calls to the Run Script action per day**. This limit resets at 12:00 AM UTC.
```

Reply:
Yes, that sounds good!